ui/de-lib/src/main/java/org/iplantc/de/analysis/client/views/AnalysisColumnModel.java

```diff
@@ -49,36 +50,69 @@
         }
     }
 
-    public static List<ColumnConfig<Analysis, ?>> createColumnConfigList(final CheckBoxSelectionModel<Analysis> checkBoxSelectionModel,
-                                                                         final AnalysesView.Appearance appearance) {
+    public static List<ColumnConfig<Analysis, ?>>
+            createColumnConfigList(final CheckBoxSelectionModel<Analysis> checkBoxSelectionModel,
+                                   final AnalysesView.Appearance appearance) {
         ColumnConfig<Analysis, Analysis> colCheckBox = checkBoxSelectionModel.getColumn();
         ColumnConfig<Analysis, Analysis> name = new ColumnConfig<>(new IdentityValueProvider<Analysis>("name"),
-                                                                                     150);
-        ColumnConfig<Analysis, Analysis> comment = new ColumnConfig<>(new IdentityValueProvider<Analysis>("description"), 30);
-        ColumnConfig<Analysis, Analysis> app = new ColumnConfig<>(new IdentityValueProvider<Analysis>("app_name"), 100);
-        ColumnConfig<Analysis, Analysis> startDate = new ColumnConfig<>(new IdentityValueProvider<Analysis>("startdate"), 125);
-        ColumnConfig<Analysis, Analysis> endDate = new ColumnConfig<>(new IdentityValueProvider<Analysis>("enddate"), 125);
+                                                                   150);
+        ColumnConfig<Analysis, Analysis> comment = new ColumnConfig<>(new IdentityValueProvider<Analysis>("description"),
+                                                                      30);
+        ColumnConfig<Analysis, Analysis> app = new ColumnConfig<>(new IdentityValueProvider<Analysis>("app_name"),
+                                                                  100);
+        ColumnConfig<Analysis, Analysis> startDate = new ColumnConfig<>(new IdentityValueProvider<Analysis>("startdate"),
+                                                                        125);
+        ColumnConfig<Analysis, Analysis> endDate = new ColumnConfig<>(new IdentityValueProvider<Analysis>("enddate"),
+                                                                      125);
         ColumnConfig<Analysis, String> status = new ColumnConfig<>(new ValueProvider<Analysis, String>() {
 
-            @Override
-            public String getValue(Analysis object) {
-                return object.getStatus();
-            }
-
-            @Override
-            public void setValue(Analysis object, String value) {
-                object.setStatus(value);
-            }
-
-            @Override
-            public String getPath() {
-                return "status";
-            }
-        }, 75);
+                                                                       @Override
```

Comment:
If the `new ColumnConfig<>(` was on a new line, its args wouldn't be indented as much.
I think our code style might also allow the args to start on the next line, which wouldn't indent them as much either.